app/models/article.rb

```diff
@@ -32,6 +32,7 @@ class Article < ApplicationRecord
   MAX_USER_MENTION_LIVE_AT = Time.utc(2021, 4, 7).freeze
   UNIQUE_URL_ERROR = "has already been taken. " \
                      "Email #{ForemInstance.email} for further details.".freeze
+  PROHIBITED_UNICODE_CHARACTERS_REGEX = /[\p{Space}&&[^ ]]/
```

Comment:
https://github.com/forem/forem/issues/4969 I'm not _actually_ sure removing non-ascii spaces (unicode whitespace) is what is needed here - the initial issue is more related to bidi override specifically and control (non-printing?) characters more generally...

If this changes (so, if the hair space \u200a is permissible, I assume that it should be) the examples in the spec will update accordingly.